package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "posterior",
+  "version": "1.0.0",
+  "description": "Backend for the Alcoding Web portal",
+  "main": "app.js",
+  "scripts": {
+    "start:dev": "nodemon",
+    "build": "rimraf ./build && tsc",
+    "start": "npm run build && node build/app.js",
+    "lint": "eslint . --ext .ts",
+    "lint-and-fix": "eslint . --ext .ts --fix",
+    "prettier-format": "prettier --config .prettierrc 'src/**/*.ts' --write",
+    "prettier-watch": "onchange 'src/**/*.ts' -- prettier --write {{changed}}"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run prettier-format && npm run lint"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pes-alcoding-club/posterior.git"
+  },
+  "keywords": [
+    "alcoding",
+    "backend"
+  ],
+  "author": "pes-alcoding-club",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/temanisparsh/posterior/issues"
+  },
+  "homepage": "https://github.com/temanisparsh/posterior#readme",
+  "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
Can we do without this dependency? Express has built in body parsing support.

Reply:
Makes sense.
I will remove this and add in the parser from express